site.properties

```diff
@@ -129,17 +125,28 @@ ldap.lastNameAttribute  = sn
 #liquibase.jdbc.username = apromore
 #liquibase.jdbc.password = MAcri
 
+impala.jdbc.driver   = com.cloudera.impala.jdbc41.Driver
+impala.jdbc.url      = ${env.IMPALA_LINK}
+impala.jdbc.username = apromore
+impala.jdbc.password = MAcri
+impala.etl.dataStore = ${env.DATA_STORE}
+
 # Database and JPA (MySQL) -- to avoid a deprecation warning in MySQL 8+, jdbc.driver = com.mysql.cj.jdbc.Driver
 
 jdbc.driver   = com.mysql.jdbc.Driver
-jdbc.url      = jdbc:mysql://localhost:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8&serverTimezone=GMT%2B10
+jdbc.url      = jdbc:mysql://apromore-mysql.apromoresso.net:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8&serverTimezone=GMT%2B10
```

Comment:
This shouldn't be the default for innocent people checking out the repo without a definition of apromore-mysql.apromoresso.net.

Reply:
@raboczi I will reverse the comments, and leave localhost:3306 as the default and apromore-mysql.apromoresso.net with an annotated comment